tests/unit/components/indicatorMenu/colourScaleIcon.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { describe, expect, test } from "vitest";
+import { interpolatePurples } from "d3-scale-chromatic";
+import { render, screen } from "@testing-library/vue";
+import { mockPinia } from "../../mocks/mockPinia";
+import ColourScaleIcon from "../../../../src/components/indicatorMenu/ColourScaleIcon.vue";
+
+describe("ColourScaleIcon", () => {
+    test("renders as expected", async () => {
+        await render(ColourScaleIcon, {
+            props: {
+                size: 24,
+                indicator: "hosp_total"
+            },
+            global: {
+                plugins: [mockPinia()]
+            }
+        });
+
+        const svg = (await screen.findByRole("presentation")) as HTMLElement;
+        expect(svg.getAttribute("height")).toBe("24");
+        expect(svg.getAttribute("width")).toBe("24");
```

Comment:
These seem like brittle tests - what if we want the icon to be a different size?